var/spack/repos/builtin/packages/nccmp/package.py

```diff
@@ -0,0 +1,30 @@
+from spack import *
+import os
+
+class Nccmp(Package):
+    """Compare NetCDF Files"""
+    homepage = "http://nccmp.sourceforge.net/"
+    url      = "http://downloads.sourceforge.net/project/nccmp/nccmp-1.8.2.0.tar.gz"
+
+    version('1.8.2.0', '81e6286d4413825aec4327e61a28a580')
+
+    depends_on('netcdf')
+
+    def install(self, spec, prefix):
+        # Configure says: F90 and F90FLAGS are replaced by FC and
+        # FCFLAGS respectively in this configure, please unset
+        # F90/F90FLAGS and set FC/FCFLAGS instead and rerun configure
+        # again.
+        os.environ['FC'] = os.environ['F90']
+        del os.environ['F90']
+        try:
+            os.environ['FCFLAGS'] = os.environ['F90FLAGS']
+            del os.environ['F90FLAGS']
+        except KeyError:    # There are no flags
```

Comment:
Not really going to be upset about either of these.  I just find a lot of Python shell-isms to be clunky, so there are a few attempts like this to make the packages more readable.  There are packagers who don't program a lot of Python, and the hope is to make it easy for them too.  Hence things like `which` and `Executable` objects.